### PR TITLE
Fix handle pre-existing ticket

### PR DESCRIPTION
**Pull request by islean**
## Description

Orders with cases should not be able to have new cases added to them. Current behaviour makes it so that the sample and cases are persisted in the database though. Now we will raise an error instead.

### Fixed

- If there is an order with the given ticket_id, an error is raised upon order submission.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by seallard — CHANGES_REQUESTED** [review]
Is this really what we want? Let me know if this is an urgent bug fix and needs to be patched. It seems very unclear to me though and maybe it would benefit to refine it business wise?

- Why do we allow customers to specify existing tickets in their orders?
- Are the customers billed separately again when they specify an existing ticket? Or not at all?
- If it is necessary to allow customers to specify existing tickets in their orders, does it not make more sense to create a separate order with the same ticket? I.e. drop the ticket uniqueness constraint in the order table

**Review by seallard — APPROVED** [review]
I think we need to change this soon. It seems strange to reuse old orders like this

File: cg/services/orders/order_service/order_service.py [hunk 1]
```diff
@@ -36,7 +38,7 @@ def get_orders(self, orders_request: OrdersRequest) -> OrdersResponse:
 
     def create_order(self, order_data: OrderIn) -> OrderResponse:
         """Creates an order and links it to the given cases."""
-        order: Order = self.store.add_order(order_data)
+        order = self.store.add_order(order_data)
```
Comment (Vince-janv) [inline cg/services/orders/order_service/order_service.py:41]: Why do we remove the type hint?

**Review by Vince-janv — APPROVED** [review]
Love it 🙌

File: cg/services/orders/order_service/order_service.py (hunk 1 above)
Comment (islean) [reply cg/services/orders/order_service/order_service.py:41]: Oh - remnant! Will bring back!

**Comment by islean** [comment]
Tested and successfully blocked submission:

image

**Comment by islean** [comment]
Deployed to cg-prod-services.

**Pull request merged**